dart-parser/src/parser/type_params.rs

```diff
@@ -39,10 +39,7 @@ where
         "type_param",
         pair(
             identifier,
-            opt(preceded(
-                tuple((spbr, tag("extends"), spbr)),
-                cut(identifier_ext),
-            )),
+            opt(preceded(tuple((spbr, tag("extends"), spbr)), cut(ty))),
```

Comment:
Note that multiple constraints are allowed (`extends (A, B)`). To be improved later.